### PR TITLE
Enable Mobile Telemetry

### PR DESCRIPTION
**Pull request by JonnavithulaGirish**
## Change log
- closes [T-33986](https://transcend.height.app/T-33986)
- Updates transcend package to 1.1.2
- 1.1.2 requires `mobileAppId` to be passed as a config paramter.
- Please refer to this [documentation](https://docs.transcend.io/docs/consent-management/mobile-consent/ios/api#version:1.1.2)

### CODE REVIEW

File: sampleSDK/sampleSDKApp.swift [hunk 1]
```diff
@@ -20,29 +21,29 @@ struct sampleSDKApp: App {
                 print("Error during web view navigation: \(error.localizedDescription)")
             }
         }
-
+        
         // Simple config object
         // expects only your airgap bundle url
-         let transcendCoreConfigSimple: TranscendCoreConfig = TranscendCoreConfig(transcendConsentUrl: "https://transcend-cdn.com/cm/63b35d96-a6db-436f-a1cf-ea93ae4be24e/airgap.js")
-
+        let transcendCoreConfigSimple: TranscendCoreConfig = TranscendCoreConfig(transcendConsentUrl: "https://transcend-cdn.com/cm-test/c7561f1c-7ec9-498c-a401-7219e3b36a8c/airgap.js")
```
Comment (anotherminh) [inline sampleSDK/sampleSDKApp.swift:27]: Which org is this?

Comment (JonnavithulaGirish) [reply sampleSDK/sampleSDKApp.swift:27]: `Consent Testing` 🙈

Comment (anotherminh) [reply sampleSDK/sampleSDKApp.swift:27]: Cool - should be fine.

Let's remove/update the references to eshopit in this repo? Maybe just use example.com instead?

Comment (anotherminh) [reply sampleSDK/sampleSDKApp.swift:27]: Or update it back to use eshopit, I don't have a strong preference either way

Comment (JonnavithulaGirish) [reply sampleSDK/sampleSDKApp.swift:27]: removed the references ✅

**Review by anotherminh — APPROVED** [review]

**Pull request merged**